.github/workflows/python-package.yml

```diff
@@ -67,7 +67,8 @@ jobs:
         # Some of the examples use Jupyter
         pip install jupyter
         # Install the version of fparser pointed to by the submodule
-        pip install external/fparser
+        # instead of the released version (from pypi).
+        # pip install external/fparser
```

Comment:
Change the comment to "If you want to install the version of .... (from pypi) then uncomment the fparser install line below".